nss-cohort-36/keahua-arboretum-pixel-pals
# report prints annexed biomes only, not inhabitants

# Description

The report prints the biomes with the id. Haven't added in functionality for animals/plants yet.


## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. start the app
1. annex a forest (or any functioning biome)
1. select 5 on the main menu
1. see the report

# Checklist:
